### PR TITLE
Add install-package to toolbox

### PR DESCRIPTION
**Pull request by bogyo210**

### CODE REVIEW

File: install-package [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+# Package installer with caching, version 2.2
+
+_show_usage_and_exit() {
+  echo -e "Package installer that caches packages and speeds up subsequent installations\n"
```
Comment (shiroyasha) [inline install-package:5]: `Package installer that caches packages` is enough.

File: install-package [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+_unpack_pkg_archive_if_available() {
+    echo ">> Unpacking package archive..."
+
+    cache restore install_package_cache
+
+    _skip_global_update
+}
+
+_archive_packages() {
```
Comment (shiroyasha) [inline install-package:41]: Why do we use `_` for function names?

**Comment by shiroyasha** [comment]
Looks great! I left several smaller questions and feedback.

Please also add this script to the list of shellchecked scripts in .semaphore/semaphore.yml.

File: install-package (hunk 2 above)
Comment (bogyo210) [reply install-package:41]: It was you or Ervin i suppose in the early days :), I just adjusted it to use the new caching system.

Comment (shiroyasha) [reply install-package:41]: I see. It was probably meant to indicate that this is `private` in the scope of the script.

There is no need for this IMHO anymore. Please remove the `_`.

Comment (bogyo210) [reply install-package]: Done.

**Comment by shiroyasha** [comment]
Looks great! Please remove the `|| true` part in the shellcheck and we are ready to ship this.

**Pull request merged**